test/libnxz.abi

```diff
@@ -80,7 +80,9 @@
     <elf-symbol name='zlibVersion' type='func-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
   </elf-function-symbols>
   <elf-variable-symbols>
+    <elf-symbol name='avg_delay' size='8' type='object-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
     <elf-symbol name='is_deflate' size='1' type='object-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
+    <elf-symbol name='metric' size='8' type='object-type' binding='global-binding' visibility='default-visibility' is-defined='yes'/>
```

Comment:
Shouldn't they be hidden? i.e. defined with `__attribute__ (visibility ("hidden")`)